src-java/rule-manager/rule-manager-implementation/src/main/java/org/openkilda/rulemanager/RuleManagerImpl.java

```diff
@@ -604,6 +601,28 @@ private List<SpeakerData> buildForwardHaRules(
         return rules;
     }
 
+    private boolean generateReverseIngressMeter(
```

Comment:
maybe consider a slightly different method name?
To make it more obvious that it is not actually generating reverseIngressMeter.
I understand that if you read this method with interrogative intonation then you will get the right meaning.. but who reads the methods in the interrogative intonation first?:)


```suggestion
    private boolean shouldGenerateReverseIngressMeter(
```

Reply:
fixed

Reply:
please don't use word should in this context.
Currently, the method name means that we hope that the Meter will be generated, but it might not happen. Better words could be: required, ought, expected.